ChurchChallenge/ChurchChallenge/View/Base.lproj/Main.storyboard

```diff
@@ -149,36 +149,46 @@
-                                <constraints>
-                                    <constraint firstAttribute="width" constant="155" id="f5r-0z-aBR"/>
-                                </constraints>
-                                <fontDescription key="fontDescription" type="boldSystem" pointSize="17"/>
-                                <nil key="textColor"/>
-                                <nil key="highlightedColor"/>
-                            </label>
-                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Date of Brith:" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" minimumFontSize="9" translatesAutoresizingMaskIntoConstraints="NO" id="zp2-xs-b4n">
-                                <rect key="frame" x="179" y="414.5" width="180" height="20.5"/>
-                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
-                                <nil key="textColor"/>
-                                <nil key="highlightedColor"/>
-                            </label>
-                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Sensitive To Force:" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="5k6-Jg-MeP">
-                                <rect key="frame" x="16" y="443" width="155" height="20.5"/>
-                                <constraints>
-                                    <constraint firstAttribute="width" constant="155" id="EA1-7e-fri"/>
-                                </constraints>
-                                <fontDescription key="fontDescription" type="boldSystem" pointSize="17"/>
-                                <nil key="textColor"/>
-                                <nil key="highlightedColor"/>
-                            </label>
-                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Date of Brith:" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" minimumFontSize="9" translatesAutoresizingMaskIntoConstraints="NO" id="RFM-Dk-zj2">
-                                <rect key="frame" x="179" y="443" width="180" height="20.5"/>
-                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
-                                <nil key="textColor"/>
-                                <nil key="highlightedColor"/>
-                            </label>
+                            <stackView opaque="NO" contentMode="scaleToFill" spacing="3" translatesAutoresizingMaskIntoConstraints="NO" id="2zG-yw-hCT">
```

Comment:
I am still not certain that using a stack view or multiple stack view is what you want here.  It seems overly complex for what you are trying to do.  It would be nice to have the labels for the data and the data line up in a nice presentation.

| | |
| -------------:|:-------------|
| Label 1: | Value 1 |
| Age: | 32 |

Reply:
Changed detail view style.